src/components/Captures/CaptureGallery.js

```diff
@@ -0,0 +1,281 @@
+          handleShowGrowerDetail={handleShowGrowerDetail}
+          handleShowCaptureDetail={handleShowCaptureDetail}
+          handleCapturePinClick={handleCapturePinClick}
+          handleGrowerMapClick={handleGrowerMapClick}
+        />
+      );
+    });
+
+  let imagePagination = (
+    <TablePagination
+      rowsPerPageOptions={[24, 96, 192, 384]}
+      component="div"
+      count={captureCount || 0}
+      rowsPerPage={rowsPerPage}
+      page={page}
+      onChangePage={handleChangePage}
+      onChangeRowsPerPage={handleChangeRowsPerPage}
+      labelRowsPerPage="Captures per page:"
+    />
+  );
+
+  return (
+    <>
+      <Grid item className={classes.body}>
+        <Grid item className={classes.bodyInner}>
+          <Grid container>
+            <Grid
+              item
+              // ref={{ refContainer }}
+              style={{
+                width: '100%',
+              }}
+            >
+              <Grid
+                container
+                justify="space-between"
+                alignItems="center"
+                className={classes.title}
+              >
+                <Typography variant="h5">
+                  {captureCount !== null &&
+                    `${countToLocaleString(captureCount)} capture${
+                      captureCount === 1 ? '' : 's'
+                    }`}
+                </Typography>
+
+                <Grid className={classes.cornerTable}>
+                  <Button
+                    variant="text"
+                    color="primary"
+                    onClick={() => setShowGallery(false)}
+                    startIcon={<TableChartIcon />}
+                  >
+                    Table View
+                  </Button>
+                  {imagePagination}
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid item>
+              <Grid
+                container
+                className={classes.wrapper}
+                spacing={1}
+                data-testid="captures-gallery"
+              >
+                {captureImageItems}
+              </Grid>
+            </Grid>
+            <Grid item container justify="flex-end" className={classes.title}>
+              {imagePagination}
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+      <SidePanel
+        onSubmit={handleSubmit}
+        submitEnabled={captures.length > 0}
+        variant="persistent"
+      />
+      {isApproveAllProcessing && (
+        <AppBar
+          position="fixed"
+          style={{
+            zIndex: 10000,
+          }}
+        >
+          <LinearProgress
+            color="primary"
+            variant="determinate"
+            value={complete}
+          />
+        </AppBar>
+      )}
+      {isApproveAllProcessing && (
+        <Modal open={true}>
+          <div></div>
+        </Modal>
+      )}
+      {false && !isApproveAllProcessing && capturesUndo.length > 0 && (
```

Comment:
Not sure about this condition...wouldn't this always prevent the section from rendering since you have 
`false && ...` ?

Reply:
In the interest of time I moved over functionality without fully untangling the parts that were essential from those that looked like they were in progress.

Since we have that code still 'in progress' in the Verify tool I'll remove it from Captures.

Reply:
@nmcharlton
Should the 'undo' and 'bulk processing be working on Verify?  It doesn't seem to be fully set up.

Reply:
No. I believe undo was broken a long time ago when we moved from individual to batch approvals and has never been fixed. There is an open issue to allow users to cancel the verification if they made a mistake.